test/new-e2e/pkg/e2e/suite.go

```diff
@@ -517,7 +518,10 @@ func (bs *BaseSuite[Env]) TearDownSuite() {
 // Unfortunately, we cannot use `s Suite[Env]` as Go is not able to match it with a struct
 // However it's able to verify the same constraint on T
 func Run[Env any, T Suite[Env]](t *testing.T, s T, options ...SuiteOption) {
-	options = append(options, WithDevMode())
```

Comment:
The `DevMode` should not have been there, it's a reliqa from the refactor merge.
Also we already have a system to handle parameters with runner/profiles which then can be set through `E2E_<param>` or through config file.
See https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/pkg/runner/parameters/const.go

Reply:
ok I will add the param devmode then instead of the lookupenv